sdk/playwrighttesting/arm-playwrighttesting/tsconfig.json

```diff
@@ -1,33 +1,18 @@
 {
+  "extends": "../../../tsconfig",
```

Comment:
@deyaaeldeen do we need to manual adopt the change in [pr](https://github.com/Azure/azure-sdk-for-js/pull/32181/files) since our codegen is not ready?

Reply:
```npx dev-tool admin migrate-package --package-name=xxx``` would do this